src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -1583,12 +1583,26 @@ public ModelEventCommand(string modelGuid, string eventName, int value = 1)
                 Value = value;
             }
 
+            [JsonConstructor]
+            public ModelEventCommand(string modelGuid, string eventName)
```

Comment:
I have added these constructor overloads back as apparently an optional parameter still constitutes a signature change. I decided against removing the optional value to avoid updating reach code and other callers of this method.